packages/react/src/ReactNodeViewRenderer.tsx

```diff
@@ -87,12 +88,14 @@ class ReactNodeView extends NodeView<React.FunctionComponent, Editor, ReactNodeV
     if (this.options.as) {
       as = this.options.as
     }
+    
+    const className = this.options.className ? ` ${this.options.className}` : ''
 
     this.renderer = new ReactRenderer(ReactNodeViewProvider, {
       editor: this.editor,
       props,
       as,
-      className: `node-${this.node.type.name}`,
+      className: `node-${this.node.type.name}` + className,
```

Comment:
Since this is already a template literal we can simply just do

```js
`node-${this.node.type.name} ${className}`.trim()
```

`.trim()` gets rid of the optional whitespace of the classname string.

Reply:
yeah, it looks better